src/utils/getUserNameFromEmail.ts

```diff
@@ -0,0 +1,4 @@
+export const getUserNameFromEmail = (email: string | null): string | null => {
```

Comment:
반환타입은 정의해주지 않아도 자동으로 추론이 되네요.

타입스크립트 코드는 바닐라 자바스크립트와 비슷하게 보일 수록 유지보수면에서 좋습니다! (자동 추론 활용)

```suggestion
export const getUserNameFromEmail = (email: string | null) => {
```